lib/libvarnishapi/vxp_lexer.c

```diff
@@ -54,10 +54,7 @@ vxp_add_token(struct vxp *vxp, unsigned tok, const char *b, const char *e)
 	t->tok = tok;
 	t->b = b;
 	t->e = e;
-	if (vxp->t != NULL)
```

Comment:
If this change goes in, please also change name to vxp_append_token(). But I think it's better to leave as is, as it then follows the same structure of libvcc (upon which this code was based), and someone familiar with libvcc won't be confused.

Reply:
I suggest we discuss this and the next comment live when we get the occasion.

Reply:
My understanding is that this comment is resolved after a brief discussion.